rules/framework.bzl

```diff
@@ -917,6 +939,11 @@ apple_framework_packaging = rule(
             allow_files = True,
             doc =
                 """Objc or Swift rules to be packed by the framework rule
+""",
+        ),
+        "library_linkopts": attr.string_list(
```

Comment:
Any reason not to rename this to `linkopts`? Mainly, it's called this in other places - it seems potentially contentious with the name of the `apple_library`

Reply:
My only thought was that I wanted to make it clear that these `linkopts` weren't forwarded on to the link operation of the underlying rule implementation and more an informative input of the higher level rule(s).

Reply:
And realizing now that because this operation passes `ctx`:
```Starlark
    link_result = linking_support.register_binary_linking_action(
        ctx,
        ...
    )
```
That having `linkopts` present "accidentally" forwards whatever `library.linkopts` has in it to the dynamic fwk's binary too.